### PR TITLE
The hands-on Jest lesson

### PR DESCRIPTION
**Pull request by haru-programming**
- [lesson](https://github.com/kenmori/handsonFrontend/blob/feature/49/jest/work.md#jest%E8%AA%B2%E9%A1%8C)
- latest commit : 5e18185

I worked on a hands-on Jest lesson.
Please review it.

### CODE REVIEW

File: index.test.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+//Q2
+
+describe("toNumber", () => {
+    test('converts "123" to equal 123', () => {
+        expect(toNumber("123", 10)).toBe(123);
+    });
+
+    test('converts "7日" to equal 7', () => {
+        expect(toNumber("7日", 10)).toBe(7);
+    });
```
Comment (kenmori) [inline index.test.js:30]: add test like to be NaN.

File: index.test.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+})
+
+
+//Q3
+
+const fruit = ["orange", "apple", "peach"];
+
+test('Check if the array contains orange', () => {
+    expect(filterOrange(fruit)).toContain("orange");
+});
```
Comment (kenmori) [inline index.test.js]: add test not include, and
Testing when another type is passed.

File: index.test.js (hunk 1 above)
Comment (haru-programming) [reply index.test.js:30]: thank you for review!
I added case of return NaN. 3dd767e

File: index.test.js (hunk 2 above)
Comment (haru-programming) [reply index.test.js]: thank you for the review!
I added a test. Is the intention of the review correct?
（I have looked into the different types of matchers, but I am not confident in my selections.）
3dd767e

**Comment by kenmori** [comment]
Approve

**Pull request merged**